shell/browser/electron_browser_main_parts.cc

```diff
@@ -384,6 +399,8 @@ void ElectronBrowserMainParts::PostDestroyThreads() {
 void ElectronBrowserMainParts::ToolkitInitialized() {
 #if BUILDFLAG(IS_LINUX)
   auto* linux_ui = ui::GetDefaultLinuxUi();
+  CHECK(linux_ui);
+  linux_ui_getter_ = std::make_unique<LinuxUiGetterImpl>();
```

Comment:
I was wondering why this class was needed when there was no direct usage seen in this PR, turns out `ui::LinuxTheme` static methods call into this instance https://chromium-review.googlesource.com/c/chromium/src/+/3853911